controller_report_xls/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# To avoid warnings in the module controller_report_xls
+cssutils==2.7.1
```

Comment:
- Does this file do anything? Requirements file should be placed at the top of the repository.
- Use `test-requirements.txt` instead, so the check to avoid requirements file doesn't complain
- Adjust title to use `ADD` and to mention the requirements file as module

Reply:
Missing the following part:

> mention the requirements file as module

I mean, the touched module in this case is the requirements file itself, not the module.

BTW, in both the requirements and the commit message you say it's to avoid warnings. I think that is not accurate because, AFAIK, if the requirement is not meet, the module fails to install. In other words, it's not a warning but an actual error.

I'd say instead:

```
[ADD] test-requirements.txt: add cssutils as requirement

Because it's required by the module `controller_report_xls`.
```

And in the requirements:

`# Required by controller_report_xls`